README.md

```diff
@@ -1,82 +1,56 @@
-# Testcontainers Cloud Go Example
+# testcontainers-cloud-go-example
 
-The current repository helps you to verify that you configured your [Testcontainers Cloud][tcc] agent correctly in your
-local environment.
+This is an example repository with a simple test confirming proper connection from Testcontainers Desktop (or the CI agent) to your [Testcontainers Cloud](https://app.testcontainers.cloud) account.
 
-## Clone the repository and run the first Testcontainer test suite
+For details on how to bootstrap Testcontainers in an actual project, please refer to the [Getting started with Testcontainers Go guide](https://github.com/testcontainers/tc-guide-getting-started-with-testcontainers-for-go).
+
+## Clone the repository and run the first Testcontainers test suite
 
 ```
 git clone https://github.com/AtomicJar/testcontainers-cloud-go-example
 cd testcontainers-cloud-go-example
-go mod download
-go test -v -count=1
 ```
 
-## Verify the agent is running
-
-✅ __Passive State__: Agent awaiting a Testcontainers test to be executed.
-
-![agent-running](./docs/passive-connection.png)
-
-✅ __Running State__: Agent connected to Testcontainers Cloud.
-
-![agent-running](./docs/active-connection.png)
-
-⚠️ __Stopped State__: Agent is stopped and will not accept connections.
-
-Please, Start the agent to continue.
-
-![agent-stopped](./docs/stopped.png)
-
-To download the agent for local usage, check the [download page here][tcc-download].
-
 ## Run the test suite
 
-`go test -v -count=1`
+Note that it's important to add the `-v` flag to the `go test` command, otherwise the output of the tests will be suppressed.
 
-### Your environment is correctly configured if
+```shell
+go test -v
```

Comment:
we had -count=1 before is it important?

Reply:
It depends :) It disables the cache for not re-running passing tests. We can keep it for avoiding undesired errors